src/platform/plugins/shared/dashboard/public/dashboard_container/component/grid/_dashboard_grid.scss

```diff
@@ -1,135 +1,27 @@
-// SASSTODO: Can't find this selector, but could break something if removed
-.react-grid-layout .gs-w {
-  z-index: auto;
-}
-
-/**
- * 1. Due to https://github.com/STRML/react-grid-layout/issues/240 we have to manually hide the resizable
- * element.
- */
-.dshLayout--viewing {
-  .react-resizable-handle {
-    display: none; /* 1 */
-  }
-}
-
-/**
- * 1. If we don't give the resizable handler a larger z index value the layout will hide it.
- */
-.dshLayout--editing {
-  .react-resizable-handle {
-    @include size($euiSizeL);
-    z-index: $euiZLevel2; /* 1 */
-    right: 0;
-    bottom: 0;
-    padding-right: $euiSizeS;
-    padding-bottom: $euiSizeS;
-  }
+.dshLayout {
```

Comment:
So much of this is getting removed, it would be amazing if we could translate the rest to emotion as a follow up. Maybe you're already working on this in https://github.com/elastic/kibana/pull/206503?

Reply:
I will be converting **at least** the `kbn-grid-layout` specific styles to Emotion in that PR - but I can definitely see what I can do for the other grid + panel styles 🤔